From 7bd9084525e4174cbb3ea5af07589c0851f77c62 Mon Sep 17 00:00:00 2001
From: Jiping Ma <jiping.ma2@windriver.com>
Date: Tue, 13 Aug 2024 01:42:23 +0000
Subject: [PATCH 2/2] ice: modify the ice driver min version to stx.3

Change the ice driver min version to stx.3 because we back ported
https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git
/commit/?id=cc2a9d6c03b804c301447326aff4cf2359867f9c (ice: Add
automatic VF reset on Tx MDD events) from linux-6.10.y.

The ice driver version should be ice-6.6.40-stx.3.

Signed-off-by: Jiping Ma <jiping.ma2@windriver.com>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 6fb76ccb9dd3..5807b310bdca 100644
--- a/Makefile
+++ b/Makefile
@@ -1227,7 +1227,7 @@ uapi-asm-generic:
 
 # KERNELRELEASE can change from a few different places, meaning version.h
 # needs to be updated, so this check is forced on all builds
-ICE_STX = "-stx.2"
+ICE_STX = "-stx.3"
 I40E_STX = "-stx.0"
 IAVF_STX = "-stx.0"
 
-- 
2.43.0

